src/libs/ReportUtils.js

```diff
@@ -1820,8 +1837,12 @@ function buildOptimisticIOUReportAction(
 
     // IOUs of type split only exist in group DMs and those don't have an iouReport so we need to delete the IOUReportID key
     if (type === CONST.IOU.REPORT_ACTION_TYPE.SPLIT) {
-        delete originalMessage.IOUReportID;
-        originalMessage.participantAccountIDs = [currentUserAccountID, ..._.pluck(participants, 'accountID')];
+        if (isPolicyExpenseChat) {
```

Comment:
cc @luacmartins @mountiny I think it makes sense to keep the `IOUReportID` here?

Reply:
Split actions don't link to a valid `IOUReportID`, so those actions shouldn't have that key. Am I missing something?

Reply:
Right, sorry my brain was fuzzy about this but it makes sense now. Will update!